### PR TITLE
[Webhook] Added Sendgrid's webhook support (added in 6.4)

### PR DESCRIPTION
**Pull request by TimoBakx**
Follow-up of https://github.com/symfony/symfony-docs/pull/19249

### CODE REVIEW

File: webhook.rst [hunk 1]
```diff
@@ -84,8 +84,14 @@ Mailer service  Parser service name
 =============== ==========================================
 Mailgun         ``mailer.webhook.request_parser.mailgun``
 Postmark        ``mailer.webhook.request_parser.postmark``
+Sendgrid        ``mailer.webhook.request_parser.sendgrid``
 =============== ==========================================
```
Comment (OskarStark) [inline webhook.rst:89]: We should add a versionadded for Mailgun and Postmark too

File: webhook.rst [hunk 2]
```diff
@@ -84,8 +84,14 @@ Mailer service  Parser service name
...
 
+.. versionadded:: 6.4
+
+    Webhook support for the Sendgrid service was introduced in Symfony 6.4
```
Comment (OskarStark) [inline webhook.rst]:
```suggestion
    Webhook support for the Sendgrid service was introduced in Symfony 6.4.
```

File: webhook.rst (hunk 1 above)
Comment (TimoBakx) [reply webhook.rst:89]: As a third column, perhaps? To have a bit better overview when more are added in the future?

**Comment by alexandre-daubois** [comment]
Oh you were fast, just saw your PR. I proposed [a different approach](https://github.com/symfony/symfony-docs/pull/19268/files), but we can keep your PR as you opened first if you want :slightly_smiling_face:

File: webhook.rst (hunk 1 above)
Comment (OskarStark) [reply webhook.rst:89]: No we usually remove the versionadded directives when merging up to a new major, so we should stick with this

Comment (TimoBakx) [reply webhook.rst:89]: The Webhook itself is covered by a 6.3 versionadded in the top of the document.

Comment (OskarStark) [reply webhook.rst:89]: Lets do this in the PR of Alex #19268

**Comment by OskarStark** [comment]
Thank you Timo.

**Pull request merged**